src/Components/App/App.js

```diff
@@ -33,8 +33,12 @@ const  App = () => {
             </main>
           )
         }}/>
+        <Route exact path='/:Activity' render={({match}) => {
```

Comment:
Great job adding this route! This will definitely help in the future with adding several pages to the application.